src/test/java/validation/cucumber/CucumberTestBase.java

```diff
@@ -62,6 +62,7 @@ public void I_send_request_with_method_name(String methodType) {
     public void i_send_request_with_specified(Integer offset) {
         response = sendRequest(Method.GET, baseQueryParameters, "hourlydate", getDateWithOffset(offset));
     }
+
     @When("I send request with specified {string} {string} {string} {string}")
```

Comment:
what is this method used for?
I think it's not only for changing longitude and latitude as the arguments are named?